### PR TITLE
fix/time_format

### LINKED ISSUE #104: fix time format in every screen

### PR DESCRIPTION
**Pull request by Hassan3Ayman**

### CODE REVIEW

File: app/src/main/java/com/cupcake/chickenmasala/ui/fragment/cuisineDishes/adapter/CuisineDishesAdapter.kt [hunk 1]
```diff
@@ -21,7 +22,7 @@ class CuisineDishesAdapter(private val listener: CuisineDishesInteractionListene
 
     override fun bindItem(binding: ItemCuisineDishesBinding, item: Recipe) {
         with(binding) {
-            textViewTime.text = item.totalTimeInMin.toString()
+            textViewTime.text = item.totalTimeInMin.toFormatTime()
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/ui/fragment/cuisineDishes/adapter/CuisineDishesAdapter.kt]: don't use here can convert in one step when use in parser

File: app/src/main/java/com/cupcake/chickenmasala/ui/fragment/details/DetailsFragment.kt [hunk 2]
```diff
@@ -85,7 +86,7 @@ class DetailsFragment : BaseFragment<FragmentDetailsBinding>() {
             textViewFoodName.text = recipe.translatedRecipeName
             textViewIngredientCount.text = recipe.ingredientCounts.toString()
             textViewCuisine.text = recipe.cuisine
-            textViewTimer.text = recipe.totalTimeInMin.toString()
+            textViewTimer.text = recipe.totalTimeInMin.toFormatTime()
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/ui/fragment/details/DetailsFragment.kt]: don't use here can convert in one step when use in parser

File: app/src/main/java/com/cupcake/chickenmasala/ui/fragment/dishes/adapter/DishesAdapter.kt [hunk 3]
```diff
@@ -32,7 +33,7 @@ class DishesAdapter(private val listener: OnItemClickListener<Recipe>) :
             cardImage.setImage(item.imageUrl)
             recipeName.text = item.translatedRecipeName
             cuisine.text = item.cuisine
-            time.text = item.totalTimeInMin.toString()
+            time.text = item.totalTimeInMin.toFormatTime()
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/ui/fragment/dishes/adapter/DishesAdapter.kt]: don't use here can convert in one step when use in parser

File: app/src/main/java/com/cupcake/chickenmasala/ui/fragment/home/adapter/HorizontalRecipeRecyclerAdapter.kt [hunk 4]
```diff
@@ -25,7 +26,7 @@ class HorizontalRecipeRecyclerAdapter(private val listener: OnItemClickListener<
             saveCardImage.setImage(item.imageUrl)
             dishName.text = item.translatedRecipeName
             cuisineName.text = item.cuisine
-            prepareTime.text = item.totalTimeInMin.toString()
+            prepareTime.text = item.totalTimeInMin.toFormatTime()
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/ui/fragment/home/adapter/HorizontalRecipeRecyclerAdapter.kt]: don't use here can convert in one step when use in parser

File: app/src/main/java/com/cupcake/chickenmasala/ui/fragment/home/adapter/VerticalRecipeRecyclerAdapter.kt [hunk 5]
```diff
@@ -23,7 +24,7 @@ class VerticalRecipeRecyclerAdapter(private val listener: OnItemClickListener<Re
         with(binding){
             cuisineName.text = item.cuisine
             recipeName.text = item.translatedRecipeName
-            recipeTime.text = item.totalTimeInMin.toString()
+            recipeTime.text = item.totalTimeInMin.toFormatTime()
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/ui/fragment/home/adapter/VerticalRecipeRecyclerAdapter.kt]: don't use here can convert in one step when use in parser

File: app/src/main/java/com/cupcake/chickenmasala/ui/fragment/search/adapter/SearchAdapter.kt [hunk 6]
```diff
@@ -54,7 +55,7 @@ class SearchAdapter(
             with(binding) {
                 textViewRecipeName.text = recipe.translatedRecipeName
                 textViewRecipeCuisine.text = recipe.cuisine
-                textViewTotalTime.text = recipe.totalTimeInMin.toString()
+                textViewTotalTime.text = recipe.totalTimeInMin.toFormatTime()
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/ui/fragment/search/adapter/SearchAdapter.kt]: don't use here can convert in one step when use in parser

File: app/src/main/java/com/cupcake/chickenmasala/utill/Extention.kt [hunk 7]
```diff
@@ -16,4 +16,15 @@ fun ImageView.setImage(url: String) {
 
 }
 
-
+fun Int.toFormatTime(): String{
+    val hours = this/60
+    val min = this % 60
+    var formatTime = ""
+    if (hours != 0){
+        formatTime = "${hours}h "
```
Comment (AmeerAmjed) [inline app/src/main/java/com/cupcake/chickenmasala/utill/Extention.kt]: when use h/m can't translated don't use hardcode, we need set app is response with Arabic .

**Review by mosayed01 — DISMISSED** [review]

**Review by mosayed01 — DISMISSED** [review]

**Pull request merged**